static/app/components/deprecatedAssigneeSelectorDropdown.tsx

```diff
@@ -113,8 +113,8 @@ type State = {
   suggestedOwners?: SuggestedOwner[] | null;
 };
 
-export class AssigneeSelectorDropdown extends Component<
-  AssigneeSelectorDropdownProps,
+export class DeprecatedAssigneeSelectorDropdown extends Component<
```

Comment:
It's nice to add a `@deprecated` block comment here with a quick note explaining what to use instead